packages/synapse-react-client/src/components/CardContainer/CardContainer.integration.test.tsx

```diff
@@ -45,34 +45,11 @@ const lastQueryRequest: QueryBundleRequest = {
   },
 }
 
-const dataWithOnePage = cloneDeep(syn16787123Json)
-dataWithOnePage.queryCount =
-  dataWithOnePage.queryResult!.queryResults.rows.length
-dataWithOnePage.queryResult!.nextPageToken = undefined
-
-const dataWithMultiplePagesFirstPage = cloneDeep(syn16787123Json)
-dataWithMultiplePagesFirstPage.queryCount =
-  dataWithMultiplePagesFirstPage.queryResult!.queryResults.rows.length * 2 + 1
-dataWithMultiplePagesFirstPage.queryResult!.nextPageToken = {
-  token: 'abcd',
-}
-
-const dataWithMultiplePagesSecondPage = cloneDeep(
-  dataWithMultiplePagesFirstPage,
-)
-dataWithMultiplePagesSecondPage.queryResult!.nextPageToken!.token = 'efgh'
-dataWithMultiplePagesSecondPage.queryResult!.queryResults.rows = [
-  ...dataWithMultiplePagesFirstPage.queryResult!.queryResults.rows.map(row => ({
-    ...row,
-    rowId: row.rowId! + DEFAULT_PAGE_SIZE,
-  })),
-]
```

Comment:
This was unnecessary, and some changed did nothing, like the `nextPageToken` logic here--table queries do not have `nextPageToken`s.